tests/core/adjust/data/plans.fmf

```diff
@@ -0,0 +1,31 @@
+discover:
+    how: fmf
+provision:
+    how: container
+prepare:
+    how: install
+    package: procps-ng
+execute:
+    how: tmt
+
+adjust:
+    prepare+:
+        package: procps
+    when: distro < centos-7
+    because: procps has been forked
+
+/fedora:
+    provision+:
+        image: fedora
+
+/centos-8:
+    provision+:
+        image: centos:8
+
+/centos-7:
+    provision+:
+        image: centos:7
+
+/centos-6:
```

Comment:
This actually makes it run the test suite on multiple images (guests) / multiple times, right? This is nice...

Reply:
Yes, exactly. We already have a similar real-life example in the shared shell tests repository:
https://src.fedoraproject.org/tests/shell/blob/master/f/plans
But those plans are intended to cover all supported shell implementations.